dashboard/app/models/hidden_stage.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: hidden_stages
+#
+#  id         :integer          not null, primary key
+#  section_id :integer
+#  stage_id   :integer
+#
+# Indexes
+#
+#  index_hidden_stages_on_section_id  (section_id)
+#  index_hidden_stages_on_stage_id    (stage_id)
+#
+
+require 'cdo/section_helpers'
```

Comment:
The file is annotated as part of the migration, but does not appear to be generated that way. As part of my renaming this, I used `rails generate SectionHiddenStages` and didn't end up with the `require 'cdo/section_helpers'` leading me to believe that I had originally copy/pasted this file from elsewhere